.github/workflows/test_protocol.yml

```diff
@@ -0,0 +1,29 @@
+name: Test BabylonNative Protocol compatibility
```

Comment:
Like we discussed offline, this isn't just testing the protocol version. Rename folder to TestCompatibility, etc.?